v7/create_error_pages/create_error_pages.py

```diff
@@ -0,0 +1,57 @@
+from nikola.plugin_categories import Task
+from nikola import utils
+
+import copy
+import os
+import os.path
+
+
+class CreateErrorPages(Task):
+    name = "create_error_pages"
+
+    def set_site(self, site):
+        super(CreateErrorPages, self).set_site(site)
+
+    def prepare_error_page(self, destination, lang, template):
+        context = {}
+
+        deps = self.site.template_system.template_deps(template)
```

Comment:
Could this be done without copy-pasting heavy bits of nikola.py? Since you already depend on future versions of Nikola, perhaps you could modify the generic page renderer to do this?